CMakeLists.txt

```diff
@@ -31,6 +31,11 @@ else()
       message("Unknown solver")
 endif()
 
+### Limits designed to make bad updates fail gracefully ###
+set(STRAIN_RATE_LIMIT "1.0e10" CACHE STRING "Limit on strain rate -- NEML may return an error if a strain rate grows beyond this value")
```

Comment:
should we also rename STRAIN_RATE_LIMIT -> NEML_STRAIN_RATE_LIMIT?

Reply:
Eh, I'm fine with our internal option not having the NEML in front of it.